databases/week2/school.sql

```diff
@@ -0,0 +1,26 @@
+--Part 2: School database......ex
+--Create a new database containing the following tables: create new schema school
+--Class: with the columns: id, name, begins (date), ends (date)
+CREATE TABLE Class (
+    class_id  INT(10), 
```

Comment:
It is better to add  AUTO_INCREMENT for the primary key.

Reply:
Thanks Ida for reviewing my HW..